spec/models/recipe_spec.rb

```diff
@@ -12,4 +24,10 @@
     it {should have_many :recipe_ingredients}
     it {should have_many(:ingredients).through(:recipe_ingredients)}
   end
+
+  describe "methods" do 
+    it 'total_cost' do 
+      expect(@spaghetti.total_cost).to eq(19)
```

Comment:
Nice job with this overall. Might be nice to see a 2nd recipe being tested; maybe one with no ingredients at all, whose `total_cost` would return `0`.